title.html

```diff
@@ -1,9 +1,9 @@
 <title>Jitsi Meet</title>
 <meta property="og:title" content="Jitsi Meet"/>
-<meta property="og:image" content="/images/jitsilogo.png?v=1"/>
```

Comment:
I typically try to avoid the ```base``` tag, as in the past, I've ran into subtle issues that were time-consuming to fix (in relation to in-page anchors, if memory serves). 

Additioally, I did not want to add them here, as I dislike how this adds a reference to a specific domain and context (although I believe that the ```href``` value of ```base``` can be relative, but why would you use ```base``` at all then?) Without the ```base``` element, you can deploy the source on any domain, with them, you'll have to manually modify the source.

I don't really see an upside to having ```base```. It is handy if you want to port an older code-base to a new context, but in general, it adds only confusion.